### DISCUSSION TITLE
Allow setting custom container labels (Unraid)

### DISCUSSION THREAD

**Discussion by Joly0** [nextcloud/all-in-one#661] [Ideas] [answered]
This is mainly an Unraid thing. Unraid uses various labels in the container run command to enable like editing the container config through the web-ui of unraid or adding icons to those containers.

It would be great to be able to set labels in the aio mastercontainer which are then used by the spawned containers aswell.

**Comment by szaimen** [comment, marked as answer]
there is this: https://github.com/nextcloud/all-in-one/tree/main/manual-install

**Reply by szaimen** [reply]
Also see https://myunraid-ru.translate.goog/nextcloud-aio/?_x_tr_sl=auto&_x_tr_tl=en&_x_tr_hl=de&_x_tr_pto=wapp
